test/globals.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+global.chai = require('chai')
+global.should = global.chai.should()
+global.sinon = require('sinon')
+global.proxyquire = require('proxyquire')
```

Comment:
I think it's typical of our projects @joefitter, but we don't usually make proxyquire gobal.